tests/unit/test_tools_changelog.py

```diff
@@ -55,9 +55,14 @@ def test_normalize_title(self):
             'Fixes #101 — Everything you can imagine is real.')
         expected = 'Fixes #101 - Everything you can imagine is real.'
         self.assertEqual(actual, expected)
-        # GreenKeeper title style
-        actual = changelog.normalize_title('Greenkeeper style 🚀')
-        expected = 'NPM update - Greenkeeper style.'
+        # # GreenKeeper title style
+        # actual = changelog.normalize_title('Greenkeeper style 🚀')
+        # expected = 'NPM update - Greenkeeper style.'
+        # self.assertEqual(actual, expected)
```

Comment:
These will not be necessary anymore. If the code is not useful, do not hesitate to delete it.